examples/fscloud/variables.tf

```diff
@@ -69,3 +69,9 @@ variable "allow_at_to_cos" {
   description = "Set rule for Activity Tracker to COS, default is true"
   default     = true
 }
+
+variable "allow_is_to_cos" {
+  type        = bool
+  description = "Set rule for IS to COS, default is true"
```

Comment:
I would just add a few words to explain what IS stands for here

Reply:
Done.